src/common/components/nav/nav.component.js

```diff
@@ -39,10 +39,22 @@ class NavController{
     this.sessionModalService = sessionModalService;
 
     this.imgDomain = envService.read('imgDomain');
-    this.navFeed = '/bin/cru/site-nav.json';
   }
 
   $onInit() {
+    //init attributes
```

Comment:
@OzzieOrca I want to do something like you have in the loading component (https://github.com/CruGlobal/give-web/blob/master/src/common/components/loading/loading.component.js#L36).  Have the defaults in the constructor.  But I wasn't able to get that to work, I always got undefined for attributes not defined in the html.  Am I misunderstanding the loading component?  Thanks